routes/api/collectors.js

```diff
@@ -0,0 +1,64 @@
+var express = require('express');
+var router = express.Router();
+var gcloud = require('gcloud');
+
+var datastore = gcloud.datastore({
```

Comment:
Let's consider how to abstract this out into a separate models js in the codebase.